services/movie.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import QuerySet
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie | None:
+    try:
+        return Movie.objects.get(id=movie_id)
+    except Movie.DoesNotExist:
+        print("Movie doesn't exist")
```

Comment:
Consider using a logging framework instead of `print` for error messages. This will provide better control over log levels and output destinations.

Reply:
You can just simply return get_object_or_404